index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="pt-br">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Analisador de números</title>
+</head>
+<body>
+  <main>
+    <h1>Analisador de números</h1>
+    <form>
+      <fieldset style="margin-bottom: 16px">
+        <label for="numero">Adicione um número entre 1 e 100</label>
+        <input type="number" id="numero" autocomplete="off">
+        <button type="button" onclick="inserirNovoNumero()">Adicionar</button>
```

Comment:
Sei que o exercício é de JS, mas não seria melhor este botão ser `type="submit"` para enviar já ao pressionar enter no input de número?

Reply:
Mesmo se for `type="submit"` ele acaba recarregando a página. Se for incorporar esse comportamento vou ter que fazer por JS, acho eu.